ReactiveCocoa/Swift/Atomic.swift

```diff
@@ -8,9 +8,56 @@
 
 import Foundation
 
+protocol MutexType: class {
```

Comment:
Maybe make this explicitly `internal`, to remind us that it's not public on purpose?